release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1046,10 +1046,12 @@ In this release, the firmware search path has been updated to copy the contents
 
 In this release, if your cluster is installed on a bare metal platform, you can scale a cluster control plane up to 5 nodes as a postinstallation task. The etcd Operator scales accordingly to account for the additional control plane nodes. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-node-scaling_recommended-etcd-practices[Node scaling for etcd].
 
-////
 [id="ocp-4-17-hcp_{context}"]
 === Hosted control planes
-////
+
+[id="ocp-4-17-hcp-tolerations_{context}"]
+==== Custom taints and tolerations (Technology Preview)
+For {hcp} on {VirtProductName}, you can now apply tolerations to hosted control plane pods by using the CLI `-tolerations` argument or by using the `hc.Spec.Tolerations` API file. This feature is available as a Technology Preview feature. For more information, see xref:../hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc#hcp-virt-taints-tolerations_hcp-distribute-workloads[Custom taints and tolerations].
```

Comment:
Is it obvious to users which CLI and commands this new argument is for? If not, it might be good to specify.